routers/web/repo/issue.go

```diff
@@ -784,6 +784,10 @@ func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles
 					}
 				}
 			}
+
+		}
+		if !strings.HasPrefix(template.Ref, "refs/") { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
```

Comment:
Why not
```suggestion
		if !strings.HasPrefix(template.Ref, git.BranchPrefix) { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
```
?

Reply:
As above.

Reply:
Because that won't work for tags. Read the associated comment.